site/composer.json

```diff
@@ -43,11 +43,11 @@
   "require-dev": {
     "dealerdirect/phpcodesniffer-composer-installer": "1.0.0",
     "php-mock/php-mock-phpunit": "2.13.0",
-    "nikic/php-parser": "v4.19.1",
-    "phpstan/phpstan": "1.10.57",
-    "phpstan/phpstan-deprecation-rules": "1.1.4",
-    "phpstan/phpstan-doctrine": "1.3.62",
-    "phpstan/phpstan-strict-rules": "1.5.2",
+    "nikic/php-parser": "v5.5.0",
```

Comment:
Can we remove our dependency of php-parser now? It will obviously remain a transitive dependency in the lock file but we shouldn't care about its version